src/js/views/signUpView.js

```diff
@@ -0,0 +1,131 @@
+export const setSignUpUI = function(){
+    let page = 0;
+    let jobCount = 1;
+    const basicInfo = document.querySelector(".basic-info");
+    const signUpAbout = document.querySelector(".signUp-about");
+    const signUpExperience = document.querySelector(".signUp-experience");
+    const prevStepBtn = document.querySelector("#previous-step-btn");
+    const nextStepBtn = document.querySelector("#next-step-btn");
+    const signUpBtn = document.querySelector("#sign-up-btn");
+    const addJobBtn = document.querySelector(".add-another-job");
+    const checkbox = document.querySelector("#current-in-role");
+    const endDateInput = document.querySelector("#end-date");
+    const presentBox = document.querySelector("#present-box");
+
+
+
+
+    
+    
+    const setUIwithPage = function(page){
+        switch(page) {
+            case 0:
+                //basicInfo: visible       signUpAbout: invisible      signUpExperience: invisible
+                basicInfo.classList.remove("invisible");
```

Comment:
Instead of making things invisible/visible, can't we just group relevant objects into components and then delete/replace them as needed like we do for clearscreen?

Reply:
We don't see the material benefits. Seems to have the same results and more effort.

Reply:
It makes this file much cleaner. Rather than having to set visibility of 6 items, you clear the screen and then display the one item